src/test/java/org/elasticsearch/indices/analyze/AnalyzeActionTests.java

```diff
@@ -191,4 +198,76 @@ private void assertTokens(AnalyzeResponse response, String ... tokens) {
     private static String indexOrAlias() {
         return randomBoolean() ? "test" : "alias";
     }
+
+    @Test
+    public void testParseAnalyzeRequest() throws Exception {
```

Comment:
Thanks for you comment.
At least, I think we need to add test using source field in request.